HelpSource/Reference/SCDocStyling.schelp

```diff
@@ -0,0 +1,46 @@
+title:: SCDoc Styling
+summary:: Help system CSS styles customization and themes
+categories:: HelpSystem
+related:: Guides/WritingHelp, Classes/SCDoc, Classes/SCDocHTMLRenderer
+
+Documentation rendered HTML reads the global style from teletype::scdoc.css::,
+but also reads teletype::frontend.css:: and teletype::custom.css:: (in that
+order) if available, to enable specific frontends and users to override the CSS.
+
+So to customise the CSS, the user can create a teletype::custom.css:: in their
+link::Classes/SCDoc#*helpTargetDir:: or at the root of any HelpSource directory
+(for example in teletype::YourExtension/HelpSource/custom.css::).
+
+code::
+// open HelpTargetDir to see these css files
+SCDoc.helpTargetDir.openOS
+::
+
+section::Colors
+
+Colors are defined as CSS variables, so that to change any color, it's
+sufficient for a user to override the relevant variable in their
+teletype::custom.css::. These variables are then used to set any color in all
+other css files of the documentation.
+
+Specific variables are not reported here, please refer to the default CSS theme
```

Comment:
Would it make sense to refer to the readme.md file around here?

Reply:
I think it is sufficient if the readme becomes apparent when navigating to the folder where the default CSS theme is located? Otherwise there could be a question why the `README` is not included here.

If you think it should be mentioned I can include it.

Reply:
Since you can make them in extensions, there ought to be some reference to it in the help files as the readme won't be next to the file. If you wouldn't mind?

Reply:
Added in https://github.com/supercollider/supercollider/pull/6095/commits/35f287b4badcab6ff9e00be3ecfb7d8a51230858